### ISSUE TITLE
Steps are not announced by Talkback (Android)

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#304]
Scenario to reproduce

Samsung Galaxy J3 Prime
Android version 7.0

Step 1 to 6 screens after splash screen

While Talkback on, using “Read from top” from “Global Context menu” Talkback starts reading from the top of the screen and moves to each step automatically and continue reading. However, it doesn’t announce what step it is reading. At the end of the step 6, it announces “Step 1 of 6 and “Next” button. Further, the same behaviour when tried swiping through each element. 

Expected result
Talkback should announce each step.

Screenshots
![screen6](https://user-images.githubusercontent.com/12256067/86131653-ead59680-bab3-11ea-92c6-3a6985df49ae.gif)

**Comment by JuliannaR** [comment]
Fixed in IOS V16.
